sympy/integrals/intpoly.py

```diff
@@ -0,0 +1,811 @@
+"""
+Module to implement integration of uni/bivariate polynomials over
+2D Polytopes(Polygons).
+
+Uses evaluation techniques as described in Chin et al(2015)[1]
+
+References
+===========
+[1] : http://dilbert.engr.ucdavis.edu/~suku/quadrature/cls-integration.pdf
+"""
+
+from __future__ import print_function, division
+
+from functools import cmp_to_key
+
+from sympy.core import S, Symbol
+
+from sympy.geometry import Segment2D
+from sympy.geometry.polygon import Polygon
+from sympy.geometry.point import Point
+from sympy.core.expr import Expr
+from sympy.abc import x, y
+from sympy.core.function import diff
+
+from sympy.polys.polytools import LC
+from sympy.polys.polytools import gcd_list
```

Comment:
The two imports could be merged as one, `from sympy.polys.polytools import LC, gcd_list`.